packages/tf-next/src/commands/delete-deployment.ts

```diff
@@ -0,0 +1,164 @@
+import { ApiGatewayV2, CloudWatchLogs, IAM, Lambda, S3 } from 'aws-sdk';
```

Comment:
So deployment state is not stored in TF, rather TF is just about the state of the platform infra.

This reminds me of an exchange we had in Notion earlier this year, and the distinction between TF for the platform vs TF for developer interface.

Reply:
Yeah, it seems like that is how it's going to be here as well. TF is not ideal for dynamic creation/deletion of infra.

Reply:
Yep agreed